src/kafka/ztf.rs

```diff
@@ -267,6 +279,7 @@ impl AlertProducer for ZtfAlertProducer {
             .set("acks", "1")
             .set("max.in.flight.requests.per.connection", "5")
             .set("retries", "3")
+            .set("debug", "broker,topic,msg")
```

Comment:
Just curious, does that have any known performance impact when set?

Reply:
I'd be surprised if there's any impact on performance because and have to enable these messages using RUST_LOG and tracing doesn't bother constructing messages unless there's a broker that's interested in consuming them. But, getting an actual answer to this question is part of #156.